samples/regex_rewrite/plugin.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START serviceextensions_regex_rewrite]
+use proxy_wasm::traits::*;
+use proxy_wasm::types::*;
+use regex::Regex;
+use std::rc::Rc;
+
+proxy_wasm::main! {{
+    proxy_wasm::set_log_level(LogLevel::Trace);
```

Comment:
I think we should update these to Critical based on feedback that this sets the maximum log level vs previously thought minimum.

Reply:
Arman, sorry for the confusion. We were wrong about the rust log level setting. This is the correct way to set the minimum log level for rust as well: this means the min log level is Trace. @leonm1, FYI

Reply:
We could expand the noop_logs example to verify + demonstrate level-setting.

Reply:
Ok thank you, I will make sure we communicate to customers accordingly.